mva/samples/data.py

```diff
@@ -139,10 +140,54 @@ def records(self,
         if include_weight:
             # data is not weighted
             weights = np.ones(rec.shape[0], dtype='f8')
-            rec = recfunctions.rec_append_fields(rec,
+#            rec = recfunctions.rec_append_fields(rec,
+            rec = rec_append_fields(rec,
                 names='weight',
-                data=weights,
-                dtypes='f8')
+                arrs=weights,
+                dtypes=np.dtype('f8'))
+        mom_branches = [
```

Comment:
The moments module is not in this commit. As far as I understand, the code won't work without it. Also maybe the moment calculation should be made optional ? Maybe with a flag in the sample class constructor ?